crates/parcel/src/requests/target_request.rs

```diff
@@ -42,22 +43,23 @@ mod package_json;
 #[derive(Debug)]
 pub struct TargetRequest {
   pub default_target_options: DefaultTargetOptions,
+  pub entry: Entry,
   pub env: Option<HashMap<String, String>>,
-  pub exclusive_target: Option<String>,
   pub mode: BuildMode,
 }
 
 impl Hash for TargetRequest {
   fn hash<H: std::hash::Hasher>(&self, state: &mut H) {
```

Comment:
Why don't we derive?

Reply:
The `env` uses `HashMap` does not derive `Hash`

Reply:
We should wrap it with a type we implement hash in then but still derive here